objects/b765cf/script.lua

```diff
@@ -144,13 +144,25 @@ function PostSetup(params)
 end
 
 function englandSnap(aidBoard)
-  local snapPoints = aidBoard.getSnapPoints()
-  local newSnapPoints = {}
-  for i,v in ipairs(snapPoints) do
-    if v.rotation.y <= 269 or v.rotation.y >= 271 then
-      table.insert(newSnapPoints,v)
+    do -- Remove aidBoard snap point
```

Comment:
what is the point of the `do end` blocks?

Reply:
`do ... end` introduces a new scope, so the variables in each block are distinct.

Reply:
is there really a concern though? wouldn't it be better to actually reuse the variable names to save on allocation?

Reply:
I've not looked in detail at the moonsharp implementation of lua, but testing here with `luac -l` to get a the bytecode for standard lua, this disappears at the bytecode level, since lua is a stack based language.